KlayGE/Plugins/Src/Render/OpenGL/OGLShaderObject.cpp

```diff
@@ -218,7 +219,9 @@ namespace
 				first = false;
 				ss.str(std::string());
 			}
-			ss << WINE_PATH << "wine ./" << d3dcompiler_wrapper_name << ".exe.so";
+			d3dcompiler_wrapper_name += ".exe.so";
+			std::string wrapper_path = ResLoader::Instance().Locate(d3dcompiler_wrapper_name);

```

Comment:
Please also modify the one in OGLESShaderObject.cpp. (I promise I will refactor this to a more code-reuse way in a near future.)